src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const COST_PER_DAY = 40;
```

Comment:
```suggestion
  const costPerDay = 40;
```

Перейменуй всі константи згідно цього зразка